designer/server/src/common/templates/layouts/page.njk

```diff
@@ -43,16 +37,16 @@
     meta: {
       items: [
         {
-          href: "https://www.gov.uk/help/privacy-notice",
-          text: "Privacy"
+          href: appPathPrefix + "/help/accessibility-statement",
+          text: "Accessibility statement"
         },
         {
-          href: "https://www.gov.uk/help/cookies",
+          href: appPathPrefix + "/help/cookies",
           text: "Cookies"
         },
         {
-          href: "https://www.gov.uk/help/accessibility-statement",
-          text: "Accessibility statement"
+          href: "https://www.gov.uk/help/privacy-notice",
+          text: "Privacy"
```

Comment:
Stuck with the GOV.UK privacy notice for now

But both **Accessibility statement** and **Cookies** pages are linked up